WordPress/src/main/res/drawable/plugin_update_available_icon.xml

```diff
@@ -0,0 +1,16 @@
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item>
+        <shape android:shape="oval">
+            <solid android:color="@color/orange_jazzy" />
```

Comment:
Nit: this color (along with the "sync" icon) kinda makes this icon look like a call-to-action button or a state icon showing that an operation is underway. If this not by design, perhaps we can revise it or ping a designer to make it more informational?

Reply:
It's by design, but I am not 100% sure this is the right color. I am currently checking with @mattmiklic about the whole icon. Let's not block the PR with this one as it might take a while to get a design review on that.

Reply:
@oguzkocer The icon looks great with one minor change; color should be `alert_yellow`.

Reply:
Great, that's an easy change. Thanks for checking it @mattmiklic!